snuba/datasets/entities/discover.py

```diff
@@ -420,41 +434,78 @@ def __init__(
             )
         )
 
-        self.__transaction_translator = transaction_translator.concat(
```

Comment:
Why aren't the translators for the errors table in the errors entity? (or events entity as of today). What is the plan there ?

Reply:
Right now, only transactions goes to the separate entity, everything else is still using the Discover entity. In a follow up, I am going to modify the table selection code to pick between 3 options (which will all be entities by then, not storages) instead of just the 2 - errors, transactions, and discover. Errors and Discover will still both use the events storage to start, but will gradually be changed over to use their new storages.